r1cs-std/src/fields/mod.rs

```diff
@@ -431,9 +431,10 @@ pub(crate) mod tests {
                 .unwrap()
         );
 
+        // a * a * a = a^3
         assert_eq!(
             a_native * &(a_native * &a_native),
-            a.pow_by_constant(cs.ns(|| "test_pow"), 3)
+            a.pow_by_constant(cs.ns(|| "test_pow"), &[3])
```

Comment:
The test is currently failing because of the repeated namespacing

Reply:
Oh woops, not quite sure why it passed on my machine.

Reply:
As a side note, is it important that this errors? It feels like something that should cause a warning at most.

Reply:
After our discussion I'm going to make namespace generation (mostly) automated, so this should not be a problem after that =)